### ISSUE TITLE
爱可可微博热门分享(2.16)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#76]
No 1. <a href="https://weibo.com/1402400261/O0PGamE2R">Sora：OpenAI的新里程碑，将文本创意转换为动态影像</a>
No 2. <a href="https://weibo.com/1402400261/O0TOKbBip">程序员的三大美德——懒惰、急躁和傲慢</a>
No 3. <a href="https://weibo.com/1402400261/O0GRODKQA">Design2Code：网页草图/截屏转代码工具，旨在将各种网页设计格式(如草图、线框图、Figma、XD等)转换为干净且响应式的HTML/CSS/JS代码。用户只需上传设计图像，Design2Code即可自动生成代码</a>
No 4. <a href="https://weibo.com/1402400261/O0TFvgf5n">视频生成模型作为世界模拟器</a>
No 5. <a href="https://weibo.com/1402400261/O0PHEiM20">Gemini 1.5 Pro：Google推出的下一代高效AI模型</a>
No 6. <a href="https://weibo.com/1402400261/O0H99EQT9">InfiAgent: 开源Agent框架，旨在支持从零开始构建Agent演示，支持代码执行、API调用、批量推断和沙盒管理</a>
No 7. <a href="https://weibo.com/1402400261/O0GcsFLSZ">用合成数据进行微调：蒸馏与自我改进</a>
No 8. <a href="https://weibo.com/1402400261/O0FNMA7EG">Langchain之类的库到底对LLM用了什么提示？用Mitmproxy进行拦截分析</a>
No 9. <a href="https://weibo.com/1402400261/O0RLzanC3">Magika：基于深度学习的新型文件类型检测工具，在超过100种内容类型的评估中，覆盖了100多种二进制和文本文件格式，实现了99%以上的精度和召回率</a>
No 10. <a href="https://weibo.com/1402400261/O0HhDBdJw">Phiz：在浏览器里进行人脸动作捕捉，使用任意移动设备或计算机上的摄像头</a>
No 11. <a href="https://weibo.com/1402400261/O0S1o1mI0">Space: 用于机器学习的统一存储解决方案，能统一处理从数据摄取到训练的整个机器学习生命周期中的数据</a>
No 12. <a href="https://weibo.com/1402400261/O0NV59pzz">[LG] Rethinking Machine Unlearning for Large Language Models ......</a>
No 13. <a href="https://weibo.com/1402400261/O0HpZmMwe">Reor：AI桌面笔记应用，能够自动连接相关的想法，回答笔记中的问题，并提供语义搜索，该项目假设思维工具默认在本地运行模型，使用 Llama.cpp 和 Transformers.js 两大开源项目，使得 LLM 和嵌入式模型都可以在本地运行</a>
No 14. <a href="https://weibo.com/1402400261/O0Hk7bemV">'Multi-Agent-GPT: 一款基于RAG和agent构建的多模态专家助手GPT，集成了文本、图像和音频等模态工具，支持本地部署和私有数据库建设' ......</a>
No 15. <a href="https://weibo.com/1402400261/O0Sj12Sw3">Pytorch-SVGRender：用于生成可微SVG向量图形的Pytorch库，支持各种不同的渲染方法</a>
No 16. <a href="https://weibo.com/1402400261/O0H4ot94v">notesGPT：由Convex、Together...</a>
No 17. <a href="https://weibo.com/1402400261/O0NWAhtAA">[LG] A Survey of Generative AI for De Novo Drug Design: New Frontiers ......</a>
No 18. <a href="https://weibo.com/1402400261/O0RMI5uce">uv：用Rust写的极快的Python软件包安装器和解析器，uv具有许多突出特点，包括与常见pip、pip-tools和virtualenv命令的即插即用替代、速度快、磁盘空间利用率高、支持高级功能等</a>
No 19. <a href="https://weibo.com/1402400261/O0EStCxcN">[LG] A Systematic Survey of Prompt Engineering in Large Language ......</a>
No 20. <a href="https://weibo.com/1402400261/O0O0HAKx6">[LG] Bayesian Strategic Classification 研究agent如何通过修改特征从学习者的分类器中获得积极的分类结果。传统假设是agent完全了解分类器，但这往往是不现实的。该研究引入了学习器部分信息披露的概念，agent对分类器有先验分布，学习器可以披露部分信息。令人惊讶的是， ......</a>
No 21. <a href="https://weibo.com/1402400261/O0NJj0VXy">[LG]《Second Order Methods for Bandit Optimization and Control》</a>
No 22. <a href="https://weibo.com/1402400261/O0GZ9hTg2">Savant: 开源高级视频/图像分析框架，用于在Nvidia栈上构建实时、流传输、高效的多媒体AI应用</a>
No 23. <a href="https://weibo.com/1402400261/NFVEo7yEd">Whishper：开源100%本地的音频转录和字幕套件，具有完整的网页用户界面，可以将任何媒体(音频、视频等)转录成文本，支持从URL转录和上传文件转录</a>
No 24. <a href="https://weibo.com/1402400261/Nz3FJz3ei">'KS的日语学习工具 v0.2 - 简易日语学习 / 视觉小说阅读辅助工具，特色包括句子分析、游戏文本提取和谷歌翻译等功能’ ......</a>
No 25. <a href="https://weibo.com/1402400261/O0NDL4QCk">早！[太阳] ​</a>
No 26. <a href="https://weibo.com/1402400261/O0I6t52yk">Poe AI 悄悄上线了计算点(compute points)访问控制机制，订阅用户每月有100万点，每月重置；非付费用户每天获得少量计算点，24小时后重置，不累积。根据引擎不同，每次对话需要的点数不同，目前GPT-4 ......</a>
No 27. <a href="https://weibo.com/1402400261/O0O4rcrvV">今日推介(第1317期)：通过响应对预测学习估计生成模型对真实世界过程的了解程度、大型语言模型推理的前提顺序问题研究、Bandit优化与控制的二阶方法、用投影梯度下降法攻击大型语言模型、用循环一致性细化多模态基础模型的提示 ......</a>
No 28. <a href="https://weibo.com/1402400261/O0L9jDoT7">晚安～ [月亮] ​</a>
No 29. <a href="https://weibo.com/1402400261/NFyOCwOZj">2023最佳科学插图集锦</a>
No 30. <a href="https://weibo.com/1402400261/NFraqEwAn">是时候承认基因不是生命蓝图了</a>
No 31. <a href="https://weibo.com/1402400261/NFkJdnIv1">Web2pdf：一个命令行工具，用于将网页转换成精美格式的PDF文件，具有批量转换、自定义样式、多列支持、页码添加、目录生成等功能，还支持自定义CSS以进一步定制PDF外观，可以帮助用户轻松地将多个网页转换为PDF文件，并在格式和样式上进行灵活的调整</a>
No 32. <a href="https://weibo.com/1402400261/O0H7m8cN5">Riallto：旨在弥合初学者和AMD最新AI引擎技术性能专家之间的差距</a>
No 33. <a href="https://weibo.com/1402400261/O0NJSaFMd">[LG]《Attacking Large Language Models with Projected Gradient Descent》</a>
No 34. <a href="https://weibo.com/1402400261/O0HvxrFhH">GPTScript：一种新的脚本语言，用于自动化与大型语言模型(LLM)OpenAI的交互，其最终目标是创建一个完全基于自然语言的编程体验</a>
No 35. <a href="https://weibo.com/1402400261/O0FQC9Smx">WildVision视觉匿名竞技场( 网页链接 )结果显示，目前胜率最高的是gpt-4-vision-preview模型，开源模型里Ilava-v1.6-34b模型排名领先 ......</a>
No 36. <a href="https://weibo.com/1402400261/O0NKwCiWK">提出一种名为CyclePrompt的技术，通过循环一致性在上下文进行cycle-supervised learning，从而提高了大型语言模型的性能，无需昂贵的微调和训练数据。//[CL]《Learning How To Ask: Cycle-Consistency Refines Prompts in Multimodal Foundation Models》</a>
No 37. <a href="https://weibo.com/1402400261/O0Hnu4WI4">autofinetune：自动化生成对话数据集，并...</a>
No 38. <a href="https://weibo.com/1402400261/O0n4IkH9i">2024 AI就业市场趋势</a>
No 39. <a href="https://weibo.com/1402400261/O0hSd8hZF">rexi：简单而强大的命令行正则表达式测试工具，专为开发人员、数据科学家和任何有兴趣直接从终端操作正则表达式的人而设计</a>
No 40. <a href="https://weibo.com/1402400261/O0NKqB4GR">[CL]《Learning How To Ask: Cycle-Consistency Refines Prompts in Multimodal Foundation Models》</a>
No 41. <a href="https://weibo.com/1402400261/O0Hjz0dlU">Lua-cad：OpenSCAD for Lua, 使用 lua 来创建三维模型</a>
No 42. <a href="https://weibo.com/1402400261/O0ELlwZAr">[LG]《World Model on Million-Length Video And Language With RingAttention》</a>
No 43. <a href="https://weibo.com/1402400261/O0HvZ3kUk">几篇论文实现代码</a>
No 44. <a href="https://weibo.com/1402400261/O0h9GfL8J">DataDreamer：一个强大的开源Python库，用于提示、合成数据生成和训练工作流，旨在简单、高效，且适用于研究。使用DataDreamer，可以轻松创建和运行多步骤的提示工作流，生成合成数据集，对模型进行训练</a>
No 45. <a href="https://weibo.com/1402400261/O0NHN581d">[LG]《Experts Don't Cheat: Learning What You Don't Know By Predicting Pairs》</a>
No 46. <a href="https://weibo.com/1402400261/O0NIW1lPF">[CL]《Premise Order Matters in Reasoning with Large Language Models》</a>
No 47. <a href="https://weibo.com/1402400261/O0NXNbX2g">[LG] Predicting User Experience on Laptops from Hardware Specifications ......</a>
No 48. <a href="https://weibo.com/1402400261/O0h7zkro2">(开源硬件)平衡立方体，可以在一个顶点上保持平衡并旋转</a>
No 49. <a href="https://weibo.com/1402400261/NFMPxCtdR">'tt-zhipin - 头头直聘，仿Boss直聘实现。SpringCloud Alibaba 构建后端，React ......</a>
No 50. <a href="https://weibo.com/1402400261/O0Hlbls5w">NanoNote：跨平台的所见即所得的 Markdown 文本编辑器，采用 Rust 语言结合 Tauri 框架进行开发</a>
